packages/shared-components/src/Forms/components/DigitalSubmissionButton.tsx

```diff
@@ -0,0 +1,59 @@
+import { TEXTS } from "@navikt/skjemadigitalisering-shared-domain";
+import { Hovedknapp } from "nav-frontend-knapper";
+import React, { useState } from "react";
+import { useAppConfig } from "../../configContext";
+import { useLanguages } from "../../context/languages";
+import { getRelevantAttachments } from "./attachmentsUtil";
+
+export interface Props {
+  form: object;
+  submission: object;
+  translations: object;
+  onError: Function;
+  onSuccess?: Function;
+}
+
+const noop = () => {};
+
+const postToSendInn = async (http, baseUrl, form, submission, translations, currentLanguage) => {
+  const translationsForPDF = currentLanguage !== "nb-NO" && translations ? translations[currentLanguage] : {};
```

Comment:
Bruk en global konstant for "nb-NO"

Reply:
Vi har faktisk ikke en konstant for nb-NO per dags dato, og stringen er spredd rundt i alle hjørner av koden vår, så tenker vi rydder opp i dette ved en annen passende anledning.